template/package.json.ejs

```diff
@@ -11,6 +11,7 @@
     "cypress:local": "CYPRESS_LOCAL=true CYPRESS_BASE_URL=http://localhost:3000 cypress",
     "db:migrate": "yarn -s prisma migrate up --experimental",
     "db:setup": "yarn db:migrate && yarn prisma generate",
+    "db:drop": "source prisma/.env && dropdb $(echo $DATABASE_NAME)",
```

Comment:
`dropdb` is a command line helper for postgres. Comes with every install - more [here](https://www.postgresql.org/docs/9.3/app-dropdb.html).

Reply:
what do you think about putting this into `scripts/dropDatabase`? If we did that we could do:

```js
import { Client } from 'pg';

const connectionString = process.env.DATABASE_URL;
const db = new Client({ connectionString });
const { database } = db.connectionParameters;

async function drop() {
  await db.connect();
  console.log(`dropping the ${database} database`);
  await db.query(`DROP DATABASE IF EXISTS ${database}`);

  await db.end();
}

drop();
```

Reply:
the big advantage here I think is that we can just use the existing `DATABASE_URL` and not have to manage another variable. we should have `pg` included as a dependency already.